src/conversion/wkb.jl

```diff
@@ -2,6 +2,115 @@
 # Licensed under the MIT License. See LICENSE in the project root.
 # ------------------------------------------------------------------
 
+function meshes2wkb(io, geom)
+  wkbtype = _wkbtype(geom)
+  write(io, htol(one(UInt8)))
+  write(io, htol(UInt32(wkbtype)))
+  geom2wkb(io, wkbtype, geom)
+end
+
+
+function _wkbtype(geometry)
+  if geometry isa Point
+    # wkbPoint 
+    return 1
+  elseif geometry isa Rope || geometry isa Ring
+    # wkbLineString
+    return 2
+  elseif geometry isa PolyArea
+    # wkbPolygon
+    return 3
+  elseif geometry isa Multi
+    # wkbMulti
+    fg = first(parent(geometry))
+    return _wkbtype(fg) + 3
+  end
+end
+
+function geom2wkb(io, wkbtype, geom)
+  if wkbtype > 3
+    multi2wkb(io, wkbtype, geom)
+  else
+    single2wkb(io, wkbtype, geom)
+  end
+end
+
+
+function single2wkb(io, wkbtype, geom)
+  # wkbPolygon
+  if wkbtype == 3
+    poly2wkb(io, [boundary(geom::PolyArea)])
+  elseif wkbtype == 2
+    coordlist = vertices(geom)
+    if typeof(geom) <: Ring
+      # wkbLineString[length+1]
+      return chainring2wkb(io, coordlist)
+    end
+    # wkbLineString
+    chainrope2wkb(io, coordlist)
+  elseif wkbtype == 1
+    # wkbPoint
+    point2wkb(io, geom)
+  else
+    throw(ErrorException("Well-Known Binary Geometry unknown: $wkbtype"))
+  end
+end
+
+function point2wkb(io, geom)
+  coordinates = CoordRefSystems.raw(coords(geom))
```

Comment:
```suggestion
  xyz = CoordRefSystems.raw(coords(geom))
```

Please try to follow the same variable names we used in the reader. That will facilitate the review and the lives of future readers.